docs/README.md

```diff
@@ -26,3 +26,13 @@ These steps should help you to build the documentation
     - `npm run doc:install`
 5. Build and open the documentation:
     - `npm run doc:dev`
+
+## Best practices
```

Comment:
These best practices are specific to the USS SDK so I think they should be moved to `src/zos_uss/README.md`